docs/src/forc/manifest_reference.md

```diff
@@ -182,3 +182,12 @@ fn main() {
   let foo_id = foo::CONTRACT_ID;
 }
 ```
+
+Since contract id is created deterministcly, building the same contract would always result with the same contract id. Since two contracts with same contract id cannot be deployed, a salt is needed to add randomness to the contract id. For each contract dependency declared under `[contract-dependencies]`, a salt can be specified by providing it with the declaration, an example can be seen below.
```

Comment:
```suggestion
Because the ID of a contract is computed deterministically, rebuilding the same contract would always result in the same contract ID. Since two contracts with same contract ID cannot be deployed on the blockchain, a "salt" factor is needed to modify the contract ID. For each contract dependency declared under `[contract-dependencies]`, `salt` can be specified. An example is shown below:
```